MetaMask/metamask-extension
# Remove my wallet account 10342

Fixes: #10342 

Explanation:  Removed the code in accordance with the instructions in the ticket

Manual testing steps:  
  - Go to 'Settings'
  -  Click on 'Contact'
  - See the section is removed from the ui 
  - see the locales and related display function are also removed
